lib/jira_wrapper.rb

```diff
@@ -98,6 +98,17 @@ def self.get_password(app_name, username)
       response.to_s.empty? ? nil : response
     end
 
+    def create_version!(release_name, project:, released: "true")
```

Comment:
as noted on invoca tools repo PR, I'm concerned about this living in the jira glue repo that has not been focused on create / edit actions, and in and of itself [in this context alone] is not very useful (since the method to attach items to the release lives over in Invoca tools)